unstable/wlr-screencopy-unstable-v1.xml

```diff
@@ -175,5 +176,31 @@
         Destroys the frame. This request can be sent at any time by the client.
       </description>
     </request>
+
+    <request name="copy_with_damage" since="2">
```

Comment:
Actually, can you add a `<!-- Version 2 additions -->` comment above this request?